File: sumall.java

import java.io.*;
import java.util.*;
public class sumall {
	public static void main(String[] args){
		try {
			System.out.println("hi");
			FileReader fr = new FileReader("sumall.in");
			BufferedReader br = new BufferedReader(fr);
			FileWriter out = new FileWriter("sumall.out");
			int n = Integer.parseInt(br.readLine());
			int sum = 0;	
				for(int x=0; x < n; x++) {
					int i = Integer.parseInt(br.readLine());
					sum += i;
				}
			out.write(String.valueOf(sum));
			out.close();
     		}
		catch(Exception e) {
			System.out.println("exception");
		}
	}
}

File: sumall.py
def main():
    try:
        filein = open("sumall.in", "r")
        sums = filein.readlines()
        sums = [int(i) for i in sums]
        sum = 0;
        for i in range(1,sums[0] + 1):
            sum += sums[i]
        print(sum)
        file = open("sumall.out", "w")
        print("file opened")
        file.write(str(sum))
        print("file wrote")
        file.close()
    except:
        print("exception");
if __name__=="__main__":
    main()
